components/RandomEpisode/RandomCard.js

```diff
@@ -1,17 +1,55 @@
-import { EpisodeImage, StyledEpisodeCard } from "../Episode/Episode.styled";
+import { useRouter } from "next/router";
+import {
+  EpisodeImage,
+  EpisodeNavButton,
+  StyledEpisodeCard,
+} from "../Episode/Episode.styled";
 import { getCoverURL } from "../Episode/EpisodeFunctions";
+import SVGIcon from "../Icons";
+import {
+  RandomEpisodeHeader,
+  RandomEpisodeNavigation,
+} from "./RandomEpisode.styled";
+
+export default function RandomCard({ coverlink, onSetRandom, onShowDetails }) {
+  const router = useRouter();
 
-export default function RandomCard({ coverlink, children }) {
   return (
     <StyledEpisodeCard>
-      {children}
+      <RandomEpisodeHeader>
+        <EpisodeNavButton
+          type="button"
+          onClick={router.back}
+          aria-label="go back"
+        >
+          <SVGIcon variant="returnIcon" width="50px" color="darkgreen" />
+        </EpisodeNavButton>
+        <h2>zufällige Folge</h2>
+      </RandomEpisodeHeader>
       <EpisodeImage
         src={getCoverURL(coverlink)}
-        alt="bla"
+        alt="Cover der zufälligen Folge"
         width={400}
         height={400}
         priority
       />
+      <RandomEpisodeNavigation>
+        <EpisodeNavButton
+          onClick={onSetRandom}
+          aria-label="get new random episode"
+        >
+          <SVGIcon variant="refresh" width="50px" color="darkred" />
+          <p>andere Folge</p>
+        </EpisodeNavButton>
+        <EpisodeNavButton
+          type="button"
+          onClick={onShowDetails}
+          aria-label="go to details page"
```

Comment:
you are mixing up english and german in aria labels and alt tags .

Reply:
you are totally right, thanks! fixed it